designs/jane/src/body.mjs

```diff
@@ -0,0 +1,227 @@
+export const body = {
+  name: 'jane.body',
+  measurements: [
+    'chest',
+    'hips',
+    'hpsToWaistBack',
+    'waistToKnee',
+    'shoulderToShoulder',
+    'neck',
+    'hpsToBust',
+    'waistToFloor',
+  ],
+  options: {
+    bodyEase: { pct: 21, min: 21, max: 50, menu: 'fit' },
+    neckDepthFront: { pct: 31, min: 25, max: 50, menu: 'style' },
+    neckDepthBack: { pct: 18, min: 15, max: 50, menu: 'style' },
+    neckWidth: { pct: 71, min: 65, max: 85, menu: 'style' },
+    shiftLength: { pct: 2, min: 0, max: 20, menu: 'style' },
+  },
+
+  draft: function draftJaneBody({
+    options,
+    Point,
+    Path,
+    points,
+    paths,
+    Snippet,
+    snippets,
+    complete,
+    sa,
+    paperless,
+    macro,
+    measurements,
+    part,
+  }) {
+    //body
+    const lengthBody =
+      (measurements.waistToKnee + measurements.hpsToWaistBack) * (1 + options.shiftLength)
+
+    const workingHip =
+      measurements.chest > measurements.hips ? measurements.chest / 2 : measurements.hips / 2
```

Comment:
Becaue it does go the knee so I am thinking it should be seat and not hips

Reply:
There will be enough ease in the seat and below with the percentages set. If you use the seat it would add a lot of unnecessary ease. The gores will widen the shift roughly starting from the hip point.